extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1080-lms-assignment.sql

```diff
@@ -39,11 +38,11 @@ BEGIN
 			ON sourceSystemDescriptor.CodeValue = lmsAssignment.SourceSystem
 
 		INNER JOIN lmsx.LMSSourceSystemDescriptor
-		ON sourceSystemDescriptor.DescriptorId  = LMSSourceSystemDescriptor.LMSSourceSystemDescriptorId
+		    ON sourceSystemDescriptor.DescriptorId  = LMSSourceSystemDescriptor.LMSSourceSystemDescriptorId
 
 		INNER JOIN edfi.Descriptor assignmentCatDescriptor
 			ON assignmentCatDescriptor.CodeValue = lmsAssignment.AssignmentCategory
-			AND assignmentCatDescriptor.Namespace = 'uri://ed-fi.org/edfilms/AssignmentCategoryDescriptor/' + lmsAssignment.SourceSystem
+			AND assignmentCatDescriptor.Namespace = 'uri://ed-fi.org/edfilms/AssignmentCategoryDescriptor/' + @SourceSystem
```

Comment:
I was worried that this might cause problems for indexing. Building the string from a constant, instead of from a value in another table, makes this cleaner anyway.